cartographer_ros/configuration_files/assets_writer_backpack_2d_ci.lua

```diff
@@ -0,0 +1,136 @@
+-- Copyright 2016 The Cartographer Authors
```

Comment:
this also looks similar to [assets_writer_backpack_2d.lua](https://github.com/googlecartographer/cartographer_ros/blob/master/cartographer_ros/configuration_files/assets_writer_backpack_2d.lua). Do you have any reason to have this independent?

Reply:
I need to remove several PointProcessors that created assets that are too large.